src/main.js

```diff
@@ -59,14 +59,17 @@ export const setParams = (event, url) => {
 };
 
 export const handleLocation = async () => {
-  const { pathname, search } = window.location;
+  const { search } = window.location;
   const urlParams = new URLSearchParams(search);
```

Comment:
In handleLocation when there is search params getExercises is called

Reply:
getExercises should be called if the name field is present. This is how it works now when clicking on exercises in a specific category